Makefile.am

```diff
@@ -176,6 +177,7 @@ linux_platform_headers = \
 	zfs/ZfsCompressedArcMeter.h
 
 linux_platform_sources = \
+	FileDescriptorMeter.c \
```

Comment:
@BenBE instead of repeating the .c and .h file in Makefile.am for each platform (except solaris AFAICT) can we instead use myhtopsources / myhtopheaders and provide solaris with similar code to 'unsupported'?

Reply:
I can implement Solaris to return `NAN`/`NAN` for that function in `SolarisPlatform.c` and leave the meter out of the list o available meters.

Good suggestion. Will update the PR accordingly.